# RFS Coordinate System and Initial Conditions.
lat0: 37.917929
lon0: -122.331798
yaw0: 0.0
X0:   88.
Y0:   -63.
Psi0: -1.5
V0:   0.0
